frontend/src/store/course/reducer.ts

```diff
@@ -41,6 +43,7 @@ const initialState: State = {
   isMentorChoosingEnabled: false,
   mentors: [],
   categories: [],
+  menteesByCourseId: null,
```

Comment:
```suggestion
  categories: [],
  menteesByCourseId:[],
```
let's be consistent

Reply:
@PavelSukhinin actually I tried in such a way but got a problem that when mentees arr is by default an empty array, and the user is a mentor and has students, the component with the list of students was rendered (the text "There no students yet" was shown for few sec) with that initial value, and didn't wait for getting the fetched array

Reply:
@kalinkaaaa14 can we change component check? something like `dataStatus === DataStatus.PENDING && !menteesByCourseId.length` -> show placeholder `There no students yet`

Reply:
@PavelSukhinin there is an issue, when the user who is not a mentor on the course viewing course page - after checking is user is a mentor, mentees will be set to [] -  and the component My users will be rendered for a plain user who is not mentor

in case of types UserDetailsResponseDto[] | null will be considered 3 situations - by default will be null, if a user is a mentor and doesn't have any students will be [], and when user is a mentor and has students UserDetailsResponseDto[]

Reply:
and then there is a possibility to render this column considering the value mentees: if null - do not render, [] - render with the placeholder, UserDetailsDto[] - render with the list of users

Reply:
should be an empty array by default. use `DataStatus` to handle it